README.md

```diff
@@ -1,4 +1,7 @@
 # Mona Python SDK
+![Mona logo](https://github.com/monalabs/mona-sdk/blob/main/mona_logo.png?raw=true)
```

Comment:
This will only work once the PR is merged (its pointing to main and the logo will be there only after the merge)

Reply:
can you please add to the PR description?